packages/core/src/components/forms/_form-group.scss

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+@import "../../common/mixins";
+
+/*
+Form groups
+
+Form groups allow for more complex forms and fields than [simple labels](#components.forms.labels.simple-labels), for example,
```

Comment:
"Form groups support more complex form controls than [simple labels], such as [control groups] or [`NumericInput`s]. They also support additional helper text to aid with user navigation."